Simulator/stm32f413.repl

```diff
@@ -148,8 +148,8 @@ spi2i2s2: SPI.STM32SPI @ sysbus 0x40003800
 
 i2s2ext: SPI.STM32SPI @ sysbus 0x40003400
 
-iwdg: Timers.STM32F4_IndependentWatchdog @ sysbus 0x40003000
-    frequency: 32000
```

Comment:
Why comment this out?

Reply:
It might be because of the version of renode you're using, although i'm not entirely sure. Try running the install script and uninstalling + reinstalling renode

Reply:
It is because the version of Renode being run but it's whatever at this point. I uncommented the lines